test_utils.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from lib import utils
+
+
+class TestUtils(unittest.TestCase):
+
+    def test_combine_dicts_with_list_values(self):
+        extended = {"1": ["1", "2", "3"],
+                    "2": ["4", "5", "6"]}
+        added = {"3": ["8", "9"],
+                 "2": ["4", "5", "6", "7"]}
+        expected = {"1": ["1", "2", "3"],
+                    "2": ["4", "5", "6", "7"],
+                    "3": ["8", "10"]}
```

Comment:
This test was actually wrong (as spotted with the 10), since sort returns None and I am essentially comparing 2 Nones. 

I reworked and corrected the test, but I am still sorting, because the combine function does not preserve the order of the elementes.